### PR TITLE
HiddenResult.py: Improve help for `contents` parameter

### PR DESCRIPTION
**Pull request by nealkarpe**
Improves the docstring so that it better describes the reason for the
existence of the `contents` parameter in the HiddenResult class.

Closes https://github.com/coala/coala/issues/4672

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by CCInCharge** [comment]
needs work a400433

Just need to update the commit message so that the first line is 50 characters or less

**Comment by CCInCharge** [comment]
ack a400433

File: coalib/results/HiddenResult.py [hunk 1]
```diff
@@ -39,8 +39,8 @@ def __init__(self,
         :param origin:
             The originating bear.
         :param contents:
-            Any object that is picklable since it will be
-            transferred across processes.
+            Used to send additional data (arbitrary python objects) besides
```
Comment (aptrishu) [inline coalib/results/HiddenResult.py]: Please make it "Any object to send additional data besides....."
would be better this way. :)

**Comment by aptrishu** [comment]
unack a400433a430b72e0abcf889e0e8e097f61a46c60

File: coalib/results/HiddenResult.py (hunk 1 above)
Comment (nealkarpe) [reply coalib/results/HiddenResult.py]: Right. That does seem more appropriate. Will make the required changes :+1:

**Comment by nealkarpe** [comment]
@aptrishu Made the suggested changes. Please review! :)

**Comment by troublemagnet** [comment]
LGTM!

**Comment by swapagarwal** [comment]
ack afc758255639df3c9dd8eef8293b594f00c33c90

**Comment by nealkarpe** [comment]
@Makman2 I think it is ready to merge. Please check it! :smiley:

**Comment by aptrishu** [comment]
@rultor merge

**Pull request merged**